### PR TITLE
2019 => develop 🥳

### PR DESCRIPTION
**Pull request by rart**

### CODE REVIEW

File: scripts/listChangedFiles.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+async function execGitCmd(args) {
+  const gitResults = await exec('git', args);
+  return gitResults
+    .trim()
+    .toString()
+    .split('\n');
+}
+
+async function listChangedFiles() {
+  const comparedBranch = process.env.CIRCLECI ? 'origin/master' : 'crafter/2019-11-ui-restructure';
```
Comment (sumerjabri) [inline scripts/listChangedFiles.js]: What's this?

File: site/website/preview-landing/index.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
```
Comment (sumerjabri) [inline site/website/preview-landing/index.xml]: Fix this copyright notice with the correct year and language.

File: site/website/preview/index.xml [hunk 3]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+  ~ Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
```
Comment (sumerjabri) [inline site/website/preview/index.xml]: Fix

File: site/website/preview/index.xml [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
   ~
   ~ This program is free software: you can redistribute it and/or modify
-  ~ it under the terms of the GNU General Public License version 3 as published by
-  ~ the Free Software Foundation.
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
```
Comment (sumerjabri) [inline site/website/preview/index.xml]: Fix

File: site/website/preview/index.xml [hunk 5]
```diff
@@ -16,8 +17,8 @@
   -->
 
 <page>
-  <internal-name>Site Dashboard</internal-name>
-  <content-type>/page/browse</content-type>	
-  <display-template>/templates/web/preview.ftl</display-template>
+  <internal-name>Preview</internal-name>
+  <content-type>/page/browse</content-type>
+  <display-template>/templates/web/legacy/preview.ftl</display-template>
```
Comment (sumerjabri) [inline site/website/preview/index.xml:21]: I thought this was changed to be `next` for new and `/` for old. No?

File: prettier.config.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
Comment (sumerjabri) [inline prettier.config.js:17]: Copyright?

File: scripts/listChangedFiles.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
+// Based on material-ui's prettier scripts
```
Comment (sumerjabri) [inline scripts/listChangedFiles.js:1]: Copyright?

File: scripts/prettier.js [hunk 8]
```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-console */
```
Comment (sumerjabri) [inline scripts/prettier.js:1]: Copyright?

File: site/website/crafter-level-descriptor.level.xml [hunk 9]
```diff
@@ -15,17 +15,12 @@
...
-      <script>/static-assets/scripts/crafter-scripts.js</script>
-    </script-group-->
-    <content-type>/acme-com/component/level-descriptor</content-type>
+<levelDescriptor xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <header xsi:nil="false">/static-assets/html/header.html</header>
+  <footer xsi:nil="false">/static-assets/html/footer.html</footer>
+  <css-group xsi:nil="true">
+    <css>/static-assets/css/style.css</css>
+  </css-group>
+  <content-type>/acme-com/component/level-descriptor</content-type>
```
Comment (sumerjabri) [inline site/website/crafter-level-descriptor.level.xml]: Did we really have ACME? Can we fix this? Something like `craftercms/...`

File: site/website/preview-landing/index.xml [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
```
Comment (sumerjabri) [inline site/website/preview-landing/index.xml]: Wrong language.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: scripts/listChangedFiles.js (hunk 1 above)
Comment (rart) [reply scripts/listChangedFiles.js]: It compares local fork/clone to the source remote to determine which files have been changed. 

Eventually, the plan is GitHub will run this process to determine if code format/quality is correct; this is why it checks if the env is the CIRCLECI. This process is not active yet so the first part of the ternary is irrelevant.

I've updated `crafter/2019-11-ui-restructure` => `crafter/develop` as this is about to become develop.

File: site/website/preview-landing/index.xml (hunk 2 above)
Comment (rart) [reply site/website/preview-landing/index.xml]: Done.

File: site/website/preview/index.xml (hunk 3 above)
Comment (rart) [reply site/website/preview/index.xml]: Done.

File: site/website/preview/index.xml (hunk 4 above)
Comment (rart) [reply site/website/preview/index.xml]: Done.

File: site/website/preview/index.xml (hunk 5 above)
Comment (rart) [reply site/website/preview/index.xml:21]: Yes. The web url was changed. This (display-template) is just the internal structure of where the FTL is located — not something that users would perceive. Notice the index.xml is located at `site/website/preview` which is what dictates the web url. 

/studio/preview => /templates/web/legacy/preview.ftl
/studio/next/preview => /templates/web/next/preview.ftl

File: prettier.config.js (hunk 6 above)
Comment (rart) [reply prettier.config.js:17]: Didn't think it made sense to copyright this file; added anyway now. 🤷🏻‍♂️

File: scripts/listChangedFiles.js (hunk 7 above)
Comment (rart) [reply scripts/listChangedFiles.js:1]: Let's discuss.

File: scripts/prettier.js (hunk 8 above)
Comment (rart) [reply scripts/prettier.js:1]: Let's discuss.

File: site/website/crafter-level-descriptor.level.xml (hunk 9 above)
Comment (rart) [reply site/website/crafter-level-descriptor.level.xml]: We did indeed 🙂 
Let's discuss: I don't see this file in develop but it comes from way back in 2015. There are two more xml files at this level not on develop that somehow are still on 2019.

File: site/website/preview-landing/index.xml (hunk 10 above)
Comment (rart) [reply site/website/preview-landing/index.xml]: Done.

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**